app/src/main/java/com/hamdan/forzenbook/theme/Color.kt

```diff
@@ -1,9 +1,35 @@
 package com.hamdan.forzenbook.theme
 
+import android.annotation.SuppressLint
+import androidx.compose.material.darkColors
+import androidx.compose.material.lightColors
 import androidx.compose.ui.graphics.Color
 
 val COLOR_LION_YELLOW = Color(0xffffe226)
 val WHITE = Color.White
```

Comment:
it seemed like a good way to avoid repeating myself in the color themeing, and if the colors need to be adjusted like
WHITE = Color(0xfffefefe)
it's easier to adjust in one spot instead of both.